File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/PlayerAttestationRenderer.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class PlayerAttestationRenderer{

	@SerializedName("botguardData")
	private BotguardData botguardData;

	@SerializedName("challenge")
	private String challenge;

	public void setBotguardData(BotguardData botguardData){
		this.botguardData = botguardData;
	}

	public BotguardData getBotguardData(){
		return botguardData;
	}

	public void setChallenge(String challenge){
		this.challenge = challenge;
	}

	public String getChallenge(){
		return challenge;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/PlayerAttestationRenderer.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class PlayerAttestationRenderer : Parcelable, Serializable {
    @SerializedName("botguardData")
    var botguardData: BotguardData? = null

    @SerializedName("challenge")
    var challenge: String? = null
    override fun toString(): String {
        return "PlayerAttestationRenderer{" +
                "botguardData = '" + botguardData + '\'' +
                ",challenge = '" + challenge + '\'' +
                "}"
    }
}